homeassistant/components/wallbox/config_flow.py

```diff
@@ -0,0 +1,114 @@
+"""Config flow for Wallbox integration."""
+import logging
+
+import requests
+import voluptuous as vol
+from wallbox import Wallbox
+
+from homeassistant import config_entries, core, exceptions
+
+from .const import DOMAIN
+
+COMPONENT_DOMAIN = DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        "station": str,
+        "username": str,
+        "password": str,
+    }
+)
+
+
+class PlaceholderHub:
```

Comment:
```suggestion
class WallboxHub:
```

Reply:
Done

Reply:
Thanks, normally when you consider a request done, hit "Resolve conversations" as well. Of course if you answer a request with a comment, then please do not resolve the conversation 😄

Reply:
yeah, but I did it locally, so marking it as resolved would not be fully accurate as of now ;-)